src/httpStats.go

```diff
@@ -18,9 +18,7 @@ type StatsData struct {
 	NumGamesSpeedrun      int
 	TimePlayedSpeedrun    string
 	NumMaxScores          int
-	TotalMaxScores        int
 	NumMaxScoresPerType   []int
-	TotalMaxScoresPerType int
```

Comment:
Alternatively, this could be  `numGamesPerType []int` I think it is more useful to see how many games were perfect out of all the games of that type than to see how many, of all perfect games, were with a certain number of players.